Feature: RTL (#2803)
adazzle/react-data-grid: README.md

- [Cell editing](https://adazzle.github.io/react-data-grid/#/common-features)
- [Cell copy / pasting](https://adazzle.github.io/react-data-grid/#/all-features)
- [Cell value dragging / filling](https://adazzle.github.io/react-data-grid/#/all-features)

> - RTL languages support. We recommend using Firefox as Chrome has a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1140374) and frozen columns do not work and [`dir:` pseudo class](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) is not supported

Reviewer: ```suggestion
- Right-to-left (RTL) text support. We recommend using Firefox as Chrome has a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1140374) with frozen columns, and the [`:dir` pseudo class](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) is not supported
```

###### `rowClass?: Maybe<(row: R) => Maybe<string>>`

##### `direction?: Maybe<'ltr' | 'rtl'>`

> This propertly sets the text direction of the grid. Setting `direction` to `rtl` has the following affects

Reviewer: ```suggestion
This property sets the text direction of the grid, it defaults to `'ltr'` (left-to-right). Setting `direction` to `'rtl'` has the following effects:
```
